Packages/tests/Basic/UTF_AnalysisBrowserTest.ipf

```diff
@@ -0,0 +1,30 @@
+#pragma TextEncoding = "UTF-8"
+#pragma rtGlobals=3 // Use modern global access method and strict wave access.
+#pragma rtFunctionErrors=1
+#pragma ModuleName=AnalysisBrowserTests
+
+// If this moves to an independend test, uncomment the following line
+//#include "unit-testing"
+
+/// @file UTF_AnalysisBrowserTest.ipf
+/// @brief __ANALYSISBROWSER_Test__ This file holds the tests for the Analysis Browser Tests
+
+static StrConstant PXP_FILENAME = "AB_LoadSweepsFromIgorData.pxp"
+
+static Function LoadSweepsFromIgor()
+
+	string win
+
+	win = AB_OpenAnalysisBrowser()
+	PathInfo home
+	REQUIRE_EQUAL_VAR(V_flag, 1)
+	PGC_SetAndActivateControl(win, "setvar_baseFolder", str=S_path)
+	PGC_SetAndActivateControl(win, "button_base_folder_scan")
+	WAVE expBrowserSel = GetExperimentBrowserGUISel()
+	expBrowserSel[0][0][0] = 81
+	PGC_SetAndActivateControl(win, "button_load_sweeps")
+	win = SWEEPBROWSER_WINDOW_TITLE + "1"
```

Comment:
The constant SWEEPBROWSER_WINDOW_TITLE does not exist anymore in latest main. It is now called SWEEPBROWSER_WINDOW_NAME. But I would actually get the list of graphs using WinList, check that it contains only one and use that.